main/.vuepress/themeConfig/nav.js

```diff
@@ -12,169 +12,84 @@ module.exports = [
       {
         text: 'Installing the Agoric SDK',
         ariaLabel: 'Installing the Agoric SDK Menu',
-        link: '/getting-started/before-using-agoric',
+        link: '/conceptual/getting-started/',
       },
       {
         text: 'Starting a Project',
         ariaLabel: 'Starting a Project Menu',
-        link: '/getting-started/start-a-project',
+        link: '/conceptual/getting-started/start-a-project',
       },
       {
         text: 'Deploying Smart Contracts',
         ariaLabel: 'Deploying Menu',
-        link: '/getting-started/deploying',
-      },
-      {
-        text: 'ERTP Introduction',
-        ariaLabel: 'ERTP Introduction Menu',
-        link: '/getting-started/ertp-introduction',
-      },
-      {
-        text: 'Zoe Introduction',
-        ariaLabel: 'Zoe Introduction Menu',
-        link: '/getting-started/intro-zoe',
-      },
-      {
-        text: 'Agoric CLI Guide and API',
-        ariaLabel: 'Agoric CLI Guide Menu',
-        link: '/guides/agoric-cli/',
-      },
-      {
-        text: 'Agoric Beta',
-        ariaLabel: 'Agoric Beta',
-        link: '/getting-started/beta',
-      },
-      {
-        text: 'Documentation Guide',
-        ariaLabel: 'Documentation Guide Menu',
-        link: '/getting-started/',
+        link: '/conceptual/getting-started/deploying',
       },
     ]
   },
   {
-    text: 'Guides',
-    ariaLabel: 'Guides',
-    items: [
-      {
-        text: 'Agoric CLI Guide and API',
-        ariaLabel: 'Agoric CLI Guide and API',
-        link: '/guides/agoric-cli/',
-      },
-      {
-        text: 'Agoric Dapps',
-        ariaLabel: 'Dapps Guide',
-        link: '/dapps/',
-      },
-      {
-        text: 'ERTP',
-        ariaLabel: 'ERTP Guide',
-        link: '/ertp/guide/'
-      },
+  text: 'Agoric Components',
```

Comment:
You're right about "components", but let's leave it alone for now. I'm not sure if it's best to break the non-components into a different bucket, or if there's a word that encompasses all the "things", or what. I want to give it some thought.... and components is good enough for now.

As for Agoric-cli, the "reference" section is really "API reference", so the CLI commands don't belong in that bucket. At least, that's how I use docs; I use API reference docs when I have an IDE open & am coding, but I want all the other content separated from the API reference. (I'm basically following the MSFT style guide here, but that's because that's what I'm used to.)

Reply:
fair enough.